<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddWord</class>
 <widget class="QDialog" name="DialogAddWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить слово</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DialogAddWord{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QLabel" name="labelNewImage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddRus">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>54</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
   <property name="text">
    <string>Русский:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAddEng">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearImage">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Без изображения</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnClearImage{
border-image: url(:/res/img/close.png);
}

#btnClearImage:hover{
border-image: url(:/res/img/closeHover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFinishAddWord">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddEng">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
   <property name="text">
    <string>English:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
   <property name="text">
    <string>Новое слово:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeImage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
   <property name="text">
    <string>Загрузить изображение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAddRus">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
